packages/tupaia-web/src/features/ExportSettings/DisplayOptionsSettings.tsx

```diff
@@ -24,8 +24,8 @@ const Checkbox = styled(BaseCheckbox)`
 
 const Group = styled(FormGroup)`
   > div {
```

Comment:
What are the `<div>`s this is selecting? If it’s just the checkboxes, then `> ${Checkbox}` might be preferable to `> div`

Alternatively (and maybe better?): Consolidate all the styling of `Checkbox` in its definition:

```css
/* const Checkbox = styled(BaseCheckbox) ... */
${FormGroup} > & {
  margin-inline-start: 0.5rem;
  margin-block-start: 0;
}
```